test/unit/spec/PrimaryAuth_spec.js

```diff
@@ -1212,7 +1232,8 @@ function (_, $, Q, OktaAuth, LoginUtil, Okta, Util, AuthContainer, PrimaryAuthFo
             password: 'pass'
           }, jasmine.any(Function));
           return tick();
-        }).then(function() {
+        })
+        .then(function() {
```

Comment:
This is intentionally **tick()** and not **waitForSpyCall** as **successSpy** will not be called as the save is prevented intentionally.